waku/v2/node/wakunode2.nim

```diff
@@ -462,10 +459,7 @@ proc mountFilter*(node: WakuNode, filterTimeout: Duration = WakuFilterTimeout) {
     for message in msg.messages:
       node.filters.notify(message, requestId) # Trigger filter handlers on a light node
       
-      # Increase message counter
-      let ctLabel = if message.contentTopic.len > 0: message.contentTopic
-                    else: "none"
-      waku_node_messages.inc(labelValues = ["filter", ctLabel])
+      waku_node_messages.inc(labelValues = ["filter"])
```

Comment:
Nothing for store?

Reply:
No, this metric is only logged for "new" messages - which we exclusively receive via `filter` or `relay` (or both, if you have a weird node setup :) )